apps/src/netsim/NetSimShard.js

```diff
@@ -38,16 +38,25 @@ var NetSimShard = module.exports = function (shardID, pubSubConfig) {
   var channel = this.pubSub.subscribe(shardID);
 
   /** @type {NetSimTable} */
-  this.nodeTable = new NetSimTable(channel, shardID, 'n');
+  this.nodeTable = new NetSimTable(shardID, 'n', {
+    channel: channel
```

Comment:
Since channel is required, suggest we make it required parameter rather than a field options which is more likely to be accidentally omitted.